.circleci/build_image.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -xe
+
+if [[ $# -ne 2 ]]; then
+  echo "Usage: $0 DOCKERFILE IMAGE_NAME"
+  exit 2
+fi
+
+DIR=`dirname $1`
+DOCKERFILE=$1
+IMAGE_NAME=$2
+
+docker build --cache-from=$IMAGE_NAME -t $IMAGE_NAME -f $DOCKERFILE $DIR
```

Comment:
Nit: I believe this should be `--cache-from=janitortechnology/ubuntu-dev`. (But do we even need to specify `--cache-from` after we `docker load -i ubuntu-dev.tar`?)

Reply:
Removed. Thanks!